addons/mail/models/ir_attachment.py

```diff
@@ -89,3 +90,13 @@ def _to_store_defaults(self):
             "type",
             "url",
         ]
+
+    def _get_author_access_token(self):
+        """Returns a scoped limited access token that indicates ownership of the attachment.
+        If verified by verify_limited_field_access_token, accessing the attachment bypasses
+        the ACLs.
+
+        :rtype: str
+        """
```

Comment:
This should indicate the flow (scope) in which it is granting access (`_has_attachments_ownership` in this case). And I'd consider renaming the method/scope/field to `attachment_ownership`/`attachment_ownership_access_token` to have a similar name in all of them.